test/hip-openmp/matrixmul_omp_copy/matrixmul.cpp

```diff
@@ -181,14 +181,14 @@ int main() {
       }
     } else printf("Unable to make initial copy\n");
   }
-
+#ifdef HIPBUGFIXED
```

Comment:
An alternative to disabling hipFree() which will leak memory would be to surround the block of code with 
#pragma omp critical 
{
...
}